rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -2534,7 +2476,7 @@ namespace rsx
 			return m_storage.m_unreleased_texture_objects;
 		}
 
-		virtual const u32 get_texture_memory_in_use() const
+		virtual const u64 get_texture_memory_in_use() const
```

Comment:
Impossible situation, we cannot access 64-bits worth of memory unless there is a counting error. Just an observation.